includes/wikia/models/NavigationModel.class.php

```diff
@@ -82,6 +82,15 @@ public function clearMemc( $key = self::WIKIA_GLOBAL_VARIABLE ) {
 		WikiaDataAccess::cachePurge(
 			$this->getMemcKey( $key )
 		);
+
+		// Purging mcache set by invocation of NavigationModel::getTree() by NavigationModel::getLocalNavigationTree();
```

Comment:
It's not used for a long time, see https://github.com/Wikia/app/commit/2f0c83fa704a81c9f2880fd8591b84c8fa540016. Let's remove it.